rest/data/commit_queue.go

```diff
@@ -88,6 +89,44 @@ func (pc *DBCommitQueueConnector) CommitQueueClearAll() (int, error) {
 	return commitqueue.ClearAllCommitQueues()
 }
 
+func (pc *DBCommitQueueConnector) IsAuthorized(ctx context.Context, username, owner, repo string) (bool, error) {
+	// In the org
+	conf, err := evergreen.GetConfig()
+	if err != nil {
+		return false, errors.Wrap(err, "can't get evergreen configuration")
+	}
+	token, err := conf.GetGithubOauthToken()
+	if err != nil {
+		return false, errors.Wrap(err, "can't get Github OAuth token from configuration")
+	}
+
+	env := evergreen.GetEnvironment()
```

Comment:
pulling these globals out of the ether (seemingly,) makes these functions pretty hard to test.

Reply:
done. And added a test :-)